### PR TITLE
Update `configure-client` section with information about ClientConfigBuilder

### LINKED ISSUE #240: Update `Configure the Polywrap Client` section
As preparation for hackathons, we need to update the `quick-start/integrate-wrappers/configure-client` page with samples of how to use the `ClientConfigBuilder`.

### PR DESCRIPTION
**Pull request by pileks**
Closes #240

### CODE REVIEW

**Review by krisbitney — APPROVED** [review]
LGTM

File: src/docs/quick-start/integrate-wrappers/configure-client.md [hunk 1]
```diff
@@ -24,6 +24,23 @@ interface ClientConfig {
...
+
+You can use the `ClientConfigBuilder` class present in `@polywrap/client-config-builder-js` to easily build the `ClientConfig` object:
+
+```typescript
+const config = new ClientConfigBuilder()
+  .addDefaults()
+  .build();
+
+// We must specify `noDefaults: true` because they are included with the `addDefaults()` method of the ClientConfigBuilder.
+const client = new PolywrapClient(config, { noDefaults: true });
```
Comment (cbrzn) [inline src/docs/quick-start/integrate-wrappers/configure-client.md:37]: i haven't tested this, but if the user does not use `.addDefaults` and does `new PolywrapClient(config)` would have the same effect?

i think it's less messy from dev exp, eventually, they will need to add the line of `.addDefaults` (but if we go with what you wrote they'll also need to remove the `{ noDefaults: true }`, right?)

Comment (pileks) [reply src/docs/quick-start/integrate-wrappers/configure-client.md:37]: Well the idea here is to show **how** to use the ClientConfigBuilder, and that includes the `noDefaults` option).

Yes, `new PolywrapClient(config)` would do the trick, but then the user is unaware of defaults being added.

I'll write something up quickly in this place that might satisfy both of us 😄

Comment (pileks) [reply src/docs/quick-start/integrate-wrappers/configure-client.md:37]: Take a look now, I added a detailed explanation with another example.

Comment (cbrzn) [reply src/docs/quick-start/integrate-wrappers/configure-client.md:38]: perfect, thank you!!

**Review by Niraj-Kamdar — APPROVED** [review]
LGTM!

**Review by cbrzn — APPROVED** [review]

**Pull request merged**